core/creators/themedui.js

```diff
@@ -384,6 +388,20 @@ CKEDITOR.replaceClass = 'ckeditor';
 		return editor;
 	}
 
+	function delayCreation( element, config, data, mode ) {
+		if ( config.delayDetached && element.isDetached() ) {
```

Comment:
Maybe more meaningful name would be useful here, fro example `delayCreationOnDetachedElement`?

Reply:
This config option is also missing API docs.

Reply:
I changed the function name.

Reply:
I added config API docs.